Anabi/Controllers/RecoveryBeneficiariesController.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Anabi.DataAccess.Abstractions.Repositories;
+using Anabi.DataAccess.Ef.DbModels;
+using Anabi.Domain.Core.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Anabi.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    public class RecoveryBeneficiariesController : BaseController
+    {
+        private readonly IGenericRepository<RecoveryBeneficiaryDb> repository;
+        public RecoveryBeneficiariesController(IGenericRepository<RecoveryBeneficiaryDb> repo)
+        {
+            repository = repo;
+        }
+
+        System.Linq.Expressions.Expression<Func<RecoveryBeneficiaryDb, RecoveryBeneficiary>> selector = c => new RecoveryBeneficiary()
+        {
+            Id = c.Id,
+            Name = c.Name            
+        };
+
+        // GET: api/RecoveryBeneficiaries
+        [HttpGet]
+        public async Task<IEnumerable<RecoveryBeneficiary>> Get()
+        {
+            try
+            {
+
+                return await repository.GetAll().Select(selector).ToListAsync();
+            }
+            catch (Exception ex)
+            {
+
+                throw;
+            }
+
+        }
+
+        // GET: api/RecoveryBeneficiaries/5
+        [HttpGet("{id}")]
+        public async Task<RecoveryBeneficiary> Get(int id)
+        {
+            try
+            {
+
+                return await repository.FindBy(p => p.Id == id).Select(selector).FirstOrDefaultAsync();
+            }
+            catch (Exception ex)
+            {
+
+                throw;
+            }
+        }
+
+        // POST: api/RecoveryBeneficiaries
+        [HttpPost]
+        public void Post([FromBody]string value)
```

Comment:
I don't think we should add methods that are not implemented. if we need functionality in a later task, the developer will add it.